.travis.yml

```diff
@@ -1,21 +0,0 @@
-language: python
-python:
-  - "2.7"
-  - "3.4"
-  - "3.5"
-
-env:
-  - DJANGO_VERSION=1.8
-  - DJANGO_VERSION=1.9
-  - DJANGO_VERSION=1.10
-
-install:
-  - cd example
-  - pip install -r requirements/base.txt
-  - pip uninstall -y -q django
-  - pip install -q Django==$DJANGO_VERSION
-  - ln -sf $(pwd)/../pyston/ $(python -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")
-
-# command to run tests
-script:
-   coverage run --omit */site-packages/*,*/migrations/*,*/lib/* manage.py test app.tests --settings=dj.settings.settings -v 2
```

Comment:
No CI? :(

Reply:
tere is still no tests, I don't know why there was travis file